napari/_vispy/vendored/volume.py

```diff
@@ -385,9 +395,22 @@
 }
 
 
+CLIPPING_PLANES_SETUP_SNIPPET = """
+    uniform vec3 clipping_plane_pos{idx};
+    uniform vec3 clipping_plane_norm{idx};
```

Comment:
```suggestion
    uniform vec3 u_clipping_plane_pos{idx};
    uniform vec3 u_clipping_plane_norm{idx};
```

for consistency, uniforms are called `u_something` across vispy